download_controller.py

```
@@ -0,0 +1,86 @@
+            print(f"{title}: {link}")
+
+        return download_links[0][1]
+
+    async def download_comic(self, comic_download_link: str) -> str:
+        async with aiohttp.ClientSession() as session:
+            async with session.get(comic_download_link) as response:
+
+                response = requests.get(comic_download_link, stream=True)
+                if response.status_code != 200:
+                    raise Exception(f"Download failed with status code {response.status_code}")
+        
```

Flake8 (7.2.0)

[warning] blank line contains whitespace

(W293)